md/processes.md

```diff
@@ -54,6 +54,8 @@ To resolve a process:
 
 ## Disable a process
 
+Disable a process can block the creation of new cases, this does not impact existing cases and the use for call activities.
```

Comment:
From version 7.4.0 on, a disabled process cannot be started by a call-activity anymore. This was a bug fix.